plugins/de.fraunhofer.ipa.ros.xtext/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
this file is not needed as there is a main one for the repository https://github.com/ipa320/ros-model/blob/master/.gitignore